completion/git-machete.completion.bash

```diff
@@ -21,7 +21,7 @@ _git_machete() {
     local reapply_opts="-f --fork-point="
     local slide_out_opts="-d --down-fork-point= -M --merge -n --no-edit-merge --no-interactive-rebase"
     local squash_opts="-f --fork-point="
-    local status_opts="--color= -L --list-commits-with-hashes -l --list-commits"
+    local status_opts="--color= -L --list-commits-with-hashes -l --list-commits -M --merge"
```

Comment:
One more thing (I hope that's my last nitpick!)... I'm not convinced that `--merge` option should really be used for this purpose, it's meaning is pretty unclear in this context :/ how about extracting e.g. `--no-detect-squash-merges` option for both `status` and `traverse` instead?

Reply:
Roger. Would you mind taking the conn on these changes in your branch and just merging from there?

I'm agnostic to the option name and documentation decisions with the skeleton of this feature in place:

* Rebase/Squash merges are detected via tree-equivalence in the "default" `git machete` mode.
* There is an escape flag to fall back to the standard merge-based detection.

Reply:
Ok! so I'm approving this branch, I'll take care of the remaining comments on my subsequent PR

Reply:
Done on https://github.com/VirtusLab/git-machete/pull/133

Reply:
Fantastic, thanks!